src/NoteParser.ts

```diff
@@ -179,7 +184,7 @@ export class Note {
     if (!ref) {
       return [];
     }
-    if (![RefType.Tag, RefType.WikiLink].includes(ref.type)) {
+    if (![RefType.Tag, RefType.WikiLink, RefType.Hyperlink].includes(ref.type)) {
```

Comment:
I am pretty sure the behavior of `_rawRangesForWord` is not expected to change, right?

Can you add some hyperlinks in the `document` tested in https://github.com/kortina/vscode-markdown-notes/blob/211ca0ac0f5fb89a9315da04ee7141e5e0ef6cf6/src/test/jest/extension.test.ts#L184 to ensure the behavior matches your expectations?

Reply:
`_rawRangesForWord` does need to change: BacklinksTreeDataProvider uses it.
The dependency looks something like this: BacklinksTreeDataProvider.getChildren uses NoteParser.searchBacklinksFor, which uses NoteParser.search, which uses Noteparser.vscodeRangesForWord, which uses NoteParser._rawRangesForWord.

Reply:
I'll add test cases in an upcoming commit.